scripts/release/bump

```diff
@@ -65,6 +65,9 @@ update_setup_py() {
   sed_in_place \
     's/^    install_requires=\["semgrep==.*"\],$/    install_requires=["semgrep=='"$release"'"],/g' \
     "$target"
+  sed_in_place \
+    's/^    version=".*",$/    version="'"$release"'",/g' \
+    "$target"
```

Comment:
how come this used to work until 0.100.0? did this code change?

Reply:
I just checked the log and originally, it was set to `0.0.0` - when we released `0.100.0`, it was updated to `0.100.0` and automation was never put around that.

Reply:
Now, it should stay up to date.